dotnet-docker-cg-code pipeline is failing to build projects for CG detection

It seems like the cg-detection pipeline isn't actually building the projects for CG detection. The error in the pipeline is shown below.

```
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file
```
[Pipeline run](https://dev.azure.com/dnceng/internal/_build/results?buildId=2409942&view=logs&j=3d756d90-2ba6-5bcd-7b25-4a1d533a722f&t=40d2c458-7009-5c59-4a8f-5ae916faa7f7) [internal link]

--- a/eng/pipelines/cg-code.yml
+++ b/eng/pipelines/cg-code.yml
@@ -8,14 +8,17 @@ trigger:
 pr: none
 
 variables:
+- template: /eng/common/templates/variables/common.yml@self
 - name: skipComponentGovernanceDetection
   value: false
+
 resources:
   repositories:
   - repository: 1ESPipelineTemplates
     type: git
     name: 1ESPipelineTemplates/1ESPipelineTemplates
     ref: refs/tags/release
+
 extends:
   template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
   parameters:

